.travis.yml

```diff
@@ -1,22 +1,41 @@
-language: python
-python:
-  - "3.6"
-  - "3.7"
-  - "3.8"
-addons:
-  apt:
-    update: true
-env:
-  - PACKAGE_NAME=pytest-confluence-report PACKAGE_VERSION=0.0.2
-before_install:
-  - sudo add-apt-repository -y ppa:duggan/bats
-  - sudo apt-get -y install bats
-install:
-  - pip install pip -U && pip install -r requirements-dev.txt -U
-script:
-  - bats test-package.bats
-  - ./analyse-source-code.sh
-after_success:
-  - coveralls
-notifications:
-  email: false
+build:
+  language: python
+  python:
+    - "3.6"
+    - "3.7"
+    - "3.8"
+  addons:
+    apt:
+      update: true
+  env:
+    - PACKAGE_NAME=pytest-confluence-report PACKAGE_VERSION=0.0.2
+  before_install:
+    - sudo add-apt-repository -y ppa:duggan/bats
+    - sudo apt-get -y install bats
+  install:
+    - pip install pip -U && pip install -r requirements-dev.txt -U
+  script:
+    - bats test-package.bats
+    - ./analyse-source-code.sh
+  after_success:
+    - coveralls
+  notifications:
+    email: false
+
+test:
+  language: python
+    python:
+      - "3.6"
+      - "3.7"
+      - "3.8"
+  addons:
+    apt:
+      update: true
+  install:
+    - pip install pip -U && pip install -r requirements-dev.txt -U
+  script:
+    pytest ./tests
```

Comment:
It is not necessary to run tests here as `./analyse-source-code.sh` script already runs the unit tests. This stage  should be responsible only for test/code analysis, including package verification as well.

I would suggest moving `script` part from `build` stage here .

Reply:
Seems I've got it. No need to separate pytest run. It's declared in analyse-source-code.sh.

Reply:
please review the changes